src/core/track-scroll-depth.ts

```diff
@@ -10,6 +10,13 @@ import { EventTimer } from './event-timer';
 const initTrackScrollDepth = () => {
 	const pageHeight = document.body.offsetHeight;
 	const intViewportHeight = window.innerHeight;
+
+	// this if statement is here to handle a bug in Firefox in Android where the innerHeight
+	// of a new tab can be 0, so we end up dividing by 0 and looping through infinity
+	if (intViewportHeight === 0) {
+		return;
```

Comment:
Maybe also logging to the commercial logger would be useful here? In case a developer is using firefox and debugging scroll depth tracking

Reply:
Good suggestion!